src/components/header/template.njk

```diff
@@ -61,8 +61,8 @@
     {% endif %}
     {% if params.navigation %}
     <button type="button" role="button" class="govuk-header__menu-button js-header-toggle" aria-controls="navigation" aria-label="Show or hide Top Level Navigation">Menu</button>
```

Comment:
Are we sure about this change? To me it seems like the label exists to compliment the `aria-controls` for the button on line 63...

Reply:
Which bit of the change are you not sure about?

Reply:
It seems from reading the source all together like the aria-label on the `ul` element exists to give it an accessible name when expanded by the button.

Since the `button` and `ul` have very similar accessible names.

Reply:
I do think the aria-label should be on the nav, rather than the ul. Perhaps though the aria-controls should point to the nav rather than the ul too?

Reply:
I think if we test this with assistive technologies it might give us an idea of the interaction.